### DISCUSSION TITLE
is renaming torrents, folders or files not case sensitive?

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#2493] [Q&A]
If try to rename a torrent from 'ABC 123' to 'abc 123' it's ignored - it's only visually renamed, but not the files or folders in the files system. I have to do it in 2 steps: 'abc 123x' and then 'abc 123' - 'x' (or any other additional character) acts as a trigger that there's really a change in content.

**Comment by pmarsden** [comment]
This is a Windows issue, it works as expected in Linux.

**Comment by ghost** [comment]
I haven't tested on Windows, but I'm experiencing it on macOS

**Comment by ckerr** [comment]
As @pmarsden says, this is likely to not be a Transmission bug. Transmission doesn't do anything to the case of the filenames before passing them along to the OS.

What filesystem is the disk that the torrent is saved on?

**Reply by ghost** [reply]
the defaul APFS (which is case-insensitive)
